apps/userdashboard/views.py

```diff
@@ -55,9 +59,21 @@ def actions(self):
     def projects_carousel(self):
         sorted_active_projects, sorted_future_projects, sorted_past_projects =\
             self.request.user.get_projects_follow_list()
-        return (list(sorted_active_projects) +
-                list(sorted_future_projects) +
-                list(sorted_past_projects))[:8]
+        display_projects = (list(sorted_active_projects) +
+                            list(sorted_future_projects) +
+                            list(sorted_past_projects))[:8]
+        projects = []
```

Comment:
I think you dont need this new list here, you can just go through the old list and add the property..